nav2_costmap_2d/include/nav2_costmap_2d/static_layer.hpp

```diff
@@ -92,6 +92,11 @@ class StaticLayer : public CostmapLayer
    */
   virtual bool isClearable() {return false;}
 
+  /**
+   * @brief If this layer can be published or not
+   */
+  virtual bool isPublishable() {return false;}
```

Comment:
Double conversion (OccGrid space -> Costmap2D space (at StaticLayer) -> OccGrid space (at publishing stage) ) is really not needed, since we might check map topic.